apps/condo/index.js

```diff
@@ -143,6 +143,53 @@ const apps = () => {
 
 /** @type {(app: import('express').Application) => void} */
 const extendExpressApp = (app) => {
+    const runtimeStats = {
+        activeRequestsIds: new Set(),
+        activeRequestsCountByType: {
+            graphql: 0,
+            oidc: 0,
+            other: 0,
+        },
+    }
+
+    app.use(function runtimeStatsMiddleware (req, res, next) {
+        const url = req.url
+        const method = (req.method || 'get').toLowerCase()
+
+        const isPost = method === 'post'
+
+        const isAdminApi = url.startsWith('/admin/api')
+        const isOidc = url.startsWith('/oidc')
+
+        /** @type {'graphql' | 'oidc' | 'other'} */
+        let requestType = 'other'
+
+        if (isPost && isAdminApi) requestType = 'graphql'
+        if (isOidc) requestType = 'oidc'
+
+        if (requestType) {
+            runtimeStats.activeRequestsIds.add(req.id)
+            runtimeStats.activeRequestsCountByType[requestType] = (runtimeStats.activeRequestsCountByType[requestType] || 0) + 1
+        }
+
+        res.on('close', () => {
+            if (requestType) {
+                runtimeStats.activeRequestsIds.delete(req.id)
+                runtimeStats.activeRequestsCountByType[requestType] = Math.max(0, runtimeStats.activeRequestsCountByType[requestType] - 1)
+            }
+        })
+
+        next()
+    })
+
+    app.get('/.well-known/runtime-stats', (req, res) => {
+        res.json({
+            activeRequestsIds: Array.from(runtimeStats.activeRequestsIds.keys()),
```

Comment:
Why do you need all ids here?

Reply:
Why not. Looks useful for investigations.

Reply:
Who should use it? I have no real scenario on my mind, since you're not logging it anywhere. And you will not investigate anything  manually, since its expiring like each 200 ms?